.cirrus.yml

```diff
@@ -8,9 +8,8 @@ build: &BUILD
     - cd rust
     - cargo test
     - cargo build --all-targets --release
-    - target/release/pjdfstest -c $CFG
-    - mkdir nobody && chown nobody nobody
-    - su -m nobody -c "target/release/pjdfstest -c $CFG -p nobody"
+    - target/release/pjdfstest -c $CFG -p /tmp
+    - su -m nobody -c "target/release/pjdfstest -c $CFG -p /tmp"
```

Comment:
I'm not sure on why it doesn't work. I'm able to successfully run it on my machine, but it fails on every Docker image that I tested.

Reply:
Yes, it is available.

Reply:
I don't know why you're getting this error.  The `-m` should allow it.  But what if you try adding a `-s /bin/sh` option to `su`?

Reply:
It works now. I used an environment variable for su args since `-s` is unsupported on FreeBSD. A little ugly, but it works!